### DISCUSSION TITLE
How to set Log retention policy.

### DISCUSSION THREAD

**Discussion by syedaqeel** [apache/dolphinscheduler#15025] [Q&A]
Hi, 

For now we are manually going to workflow instances Under all projects and deleting old jobs but this will be donkey work if after all the automation we manually delete logs. So I've been wondering if their is any smarter way to manage logs or maybe set a log retention policy.

**Comment by ruanwenjun** [comment]
There is not  a way to auto delete instance log, are you interest to support workflow instance retention policy?

**Reply by syedaqeel** [reply]
Yes I am interested to support this, can you share some background on this ?
